pkg/kubelet/kubelet.go

```diff
@@ -148,7 +149,8 @@ const (
 	evictionMonitoringPeriod = time.Second * 10
 
 	// The path in containers' filesystems where the hosts file is mounted.
-	etcHostsPath = "/etc/hosts"
+	linuxEtcHostsPath   = "/etc/hosts"
+	windowsEtcHostsPath = "C:\\Windows\\System32\\drivers\\etc\\hosts"
```

Comment:
is the drive/path to the windows installation a constant? I thought both were changeable

Reply:
should probably use https://godoc.org/golang.org/x/sys/windows#GetSystemDirectory

Reply:
Inside a Windows container, they're constant.